src/main/java/seedu/address/logic/commands/ExportCommand.java

```diff
@@ -0,0 +1,73 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.io.IOException;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+/**
+ * Exports the chosen details of the filtered participants to a CSV file.
+ */
+public class ExportCommand extends Command {
+
+    public static final String COMMAND_WORD = "export";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Exports the details of all persons identified "
+            + "by the command flags. "
+            + "Flags: n/ (name), p/ (phone), e/ (email), a/ (address)\n"
+            + "Example: " + COMMAND_WORD + " n/ p/";
```

Comment:
Make the phrasing more clear, so users know that only the fields with the flags given will be exported.